Authentication.md

```diff
@@ -56,6 +56,8 @@ You can login as any sudoer or member of the admin_console_group, but Change Pas
 
 * Status Code 403 (unauthorized) is used for unathenticated. (Some use 401.)
 * Authentication is performed by the uswgi app as a wrapper to CMDSRV.
+* uswgi makes AUTHENTICATE request to CMDSRV as uwsgi has no access to /etc/shadow
+* AUTHENTICATE command is blocked calls by front end to prevent password guessing
```

Comment:
Is there an English grammar issue or missing word below, that can be quickly cleared up if possible?

> * AUTHENTICATE command is blocked calls by front end to prevent password guessing